### ISSUE TITLE
Cannot send serializable classes in Camel messages

### ISSUE THREAD

**Issue by jvz** [mpilone/hazelcastmq#12]
Using the normal Hazelcast component, I can send any message body that's Serializable. Using HazelcastMQ in Camel, I get the following exception when I try to send a Serializable message body:

```
Caused by: org.apache.camel.RuntimeCamelException: Unsupported message body type: com.example.DataObject
    at org.mpilone.hazelcastmq.camel.DefaultMessageConverter.fromCamelMessage(DefaultMessageConverter.java:64)
    at org.mpilone.hazelcastmq.camel.HazelcastMQCamelProducer.process(HazelcastMQCamelProducer.java:102)
    at org.apache.camel.util.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:61)
    at org.apache.camel.processor.CamelInternalProcessor.process(CamelInternalProcessor.java:190)
    at org.apache.camel.util.AsyncProcessorHelper.process(AsyncProcessorHelper.java:109)
    at org.apache.camel.processor.UnitOfWorkProducer.process(UnitOfWorkProducer.java:68)
    at org.apache.camel.impl.ProducerCache$2.doInProducer(ProducerCache.java:412)
    at org.apache.camel.impl.ProducerCache$2.doInProducer(ProducerCache.java:380)
    at org.apache.camel.impl.ProducerCache.doInProducer(ProducerCache.java:270)
    at org.apache.camel.impl.ProducerCache.sendExchange(ProducerCache.java:380)
    at org.apache.camel.impl.ProducerCache.send(ProducerCache.java:221)
    at org.apache.camel.impl.DefaultProducerTemplate.send(DefaultProducerTemplate.java:124)
    at org.apache.camel.impl.DefaultProducerTemplate.sendBody(DefaultProducerTemplate.java:137)
```

DataObject is pretty simple:

``` java
public class DataObject implements Serializable {
    public UUID id;
    public String name;
}
```

Edit: upon further investigation, even though camel-hazelcast does accept non-serializable objects, it doesn't seem to transmit them. Tested with `hazelcast:seda`.

**Comment by mpilone** [comment]
The HzMq Camel configuration will use the DefaultMessageConverter to convert from Camel messages to HazelcastMQ messages. The default converter only supports text and byte[] message bodies. You have two options:
1. Serialize the object in the route before sending it to the HzMq endpoint.
   or
2. Implement your own MessageConverter that performs object serialization.

Either solution should be fairly trivial. Take a look at o.m.h.camel.DefaultMessageConverter for an example of how to implement the converter as well as the documentation on the limitations of that implementation.

**Comment by mpilone** [comment]
Thanks for the patch. It is merged into the 1.5 develop branch.
